### PR TITLE
[16.0][MIG] purchase_order_responsible

### PR DESCRIPTION
**Pull request by carmenbianca**
## Odoo task (if applicable)

https://gestion.coopiteasy.be/web#id=11104&action=475&active_id=492&model=project.task&view_type=form&menu_id=

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

**Review by victor-champonnois — APPROVED** [review]
Functional review OK
- set responsible replaces the create_uid field : OK
- replace the mail_follower : OK
- adds the referent name in the purchase order pdf report : OK

File: purchase_order_responsible/views/purchase_order.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.view_purchase_order_filter" />
+        <field name="arch" type="xml">
+            <field name="user_id" position="after">
+                <field name="supervisor_id" />
+            </field>
+            <filter name="my_purchases" position="after">
+                <filter
+                    name="my_responsible"
+                    string="My Responsible"
```
Comment (victor-champonnois) [inline purchase_order_responsible/views/purchase_order.xml:35]: "My responsible" ? Is is a correct formulation ?

**Review by victor-champonnois — COMMENTED** [review]
Code review OK (just a comment on a formulation)

**Review by robinkeunen — APPROVED** [review]
I reckon this could be submitted to the OCA

**Pull request merged**